packages/service-library/src/servicelib/rabbitmq/_client.py

```diff
@@ -82,7 +82,9 @@ async def _on_message(
                 if not await message_handler(message.body):
                     await _safe_nack(message_handler, max_retries_upon_error, message)
         except Exception:  # pylint: disable=broad-exception-caught
-            await _safe_nack(message_handler, max_retries_upon_error, message)
+            _logger.exception("Exception raised when handling message")
+            with log_catch(_logger):
```

Comment:
WARNING! not it reraises and before you did not
```python
with log_catch(_logger, reraise=False):
    await _safe_nack(...)
```

Reply:
Thanks for catching this. Indeed it should not reraise! I will fix